Logger/FileLogger.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private readonly string FilePath;
```

Comment:
This is a field, but named like a property. 
```suggestion
        private string FilePath { get; }
```